tests/components/local_calendar/test_calendar.py

```diff
@@ -873,6 +873,7 @@ async def test_create_event_service(
             "start_date_time": start_date_time,
             "end_date_time": end_date_time,
             "summary": "Bastille Day Party",
+            "location": "Test Location",
```

Comment:
The tests are not passing yet. My impression is that 'location' is not exposed through the rest API, so what I would suggest is:
- Revert the change to event_fields
- Add a test that looks at the entity attributes instead similar to `test_active_event`

Reply:
Sorry I saw your message too late. I changed that here. 

https://github.com/home-assistant/core/blob/a4fc17271ed39547b09e63851a1490af00f83e8f/homeassistant/components/local_calendar/calendar.py#L199

Please let me know if now it's ok.

Reply:
OK, yes, makes sense.

Reply:
Mark not draft when ready for review.